project/crafting/ongoing_projects/index.html

```diff
@@ -1,2 +1,3 @@
+<project-anchor work_status='ongoing' type='crafting' link='2025_10_05.belt_bag' title='Belt Bag'></project-anchor>
```

Comment:
Not sure if this is meant to be here or if you "deleted" the project before.  This index is automatically made by the Makefiles as well.  So long as a project is in the "ongoing_projects" folder, it will create an index element like this.